addons/mysql-cluster/templates/cluster.yaml

```diff
@@ -5,16 +5,53 @@ metadata:
   labels: {{ include "kblib.clusterLabels" . | nindent 4 }}
   annotations:
     kubeblocks.io/extra-env: '{ "MYSQL_TOPOLOGY_MODE": {{ default "standalone" .Values.mode | quote }} }'
+    kubeblocks.io/enabled-pod-ordinal-svc: mysql,proxysql
 spec:
+
   clusterVersionRef: {{ .Values.version }}
   terminationPolicy: {{ .Values.extra.terminationPolicy }}
   {{- include "kblib.affinity" . | indent 2 }}
   clusterDefinitionRef: mysql # ref clusterdefinition.name
   componentSpecs:
     - name: mysql
+      replicas: {{ .Values.replicas }}
       componentDefRef: mysql # ref clusterdefinition componentDefs.name
-      {{- include "kblib.componentMonitor" . | indent 6 }}
+      {{- $clusterversion := .Values.version }}
```

Comment:
cv and componentDefRef fields are not used and we should remove them and test how it works when referring differnt cmpds.